fn main() {
    let x = 5;
    println!("xの値: {}", x);

    // 以下はエラーとなる
    // x = 6;
}
